README.md

```diff
@@ -8,6 +8,8 @@
 [![Test Coverage](https://codecov.io/github/Workiva/over_react/coverage.svg?branch=master)](https://codecov.io/github/Workiva/over_react?branch=master)
 
 > A library for building statically-typed React UI components using Dart.
+>
+> This library also exposes OverReact_Redux, which has [its own documentation](doc/over_react_redux_documentation.md).
```

Comment:
There's no underscore in the name. Italics might also be nice to help separate it from the rest of the text

```suggestion
> This library also exposes _OverReact Redux_, which has [its own documentation](doc/over_react_redux_documentation.md).
```